packages/@dcl/js-runtime/sdk.d.ts

```diff
@@ -0,0 +1,5 @@
+
+// @internal
+declare module '~sdk/all-composites' {
+    export const compositeFromLoader: Record<string, Uint8Array | string>
```

Comment:
I think that importing the json as an object will be _way_ more efficient than the serialized versions. Not in final bundle size, but yes in processing and allocations

```suggestion
    export const compositeFromLoader: Record<string, CompositeType>
```